### PR TITLE
Clarify Z-Wave control panel docs

### PR DESCRIPTION
**Pull request by adamjernst**
- Make it clear exactly how to get to the control panel.
- Add a link to Z-Wave setup. (This confused me since I arrived at this page before I set up Z-Wave.)
- Demoted very specific note about renaming Z-Wave devices.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good!

I wonder, should we mention that we are currently aiming for ozw and this integration is considered legacy?

**Comment by adamjernst** [comment]
> I wonder, should we mention that we are currently aiming for ozw and this integration is considered legacy?

@frenck My personal view is that we should not. ozw is still in beta, and users shouldn't need to know about beta software until it's ready (unless they want to).

I would *also* advocate for taking ozw out of beta as soon as possible, and renaming the existing Z-Wave integration to indicate that it is legacy. But that's not my call!

**Comment by frenck** [comment]
Well `ozw` isn't ready for that yet. However, the `zwave` runs on an older OpenZWave and doesn't support newer devices.

I'm not saying we should recommend it. But we could at least have the decency to mention it to a user.

**Comment by adamjernst** [comment]
In that case I'll happily add a note about it.

**Comment by adamjernst** [comment]
Done 👍

**Comment by pauldegraaff** [comment]
No it is not a duplicate.

File: source/_docs/z-wave/control-panel.markdown [hunk 1]
```diff
@@ -3,14 +3,15 @@ title: "Z-Wave Control Panel"
...
 
-<div class='note'>
+The Z-Wave control panel is available via **Configuration** &rarr; **Integrations** &rarr; **Z-Wave** &rarr; **Configure**.
 
-  If you don't see the **Configuration** menu on the menubar, where you'll find the Z-Wave menu, [see here](/integrations/config/).
+* **No Z-Wave integration?** Follow the [set up instructions](/docs/z-wave/installation/) to enable it.
+* **Missing the *Configuration* button?** See the [configuration documentation](/integrations/config/) for instructions on enabling it.
 
+<div class='note'>
+The current Z-Wave integration will eventually be deprecated and replaced with a <a href="https://developers.home-assistant.io/blog/2020/02/04/new-zwave/">new implementation based on OZWDaemon</a>; it's currently in beta, and you can <a href="https://www.home-assistant.io/integrations/ozw/">try it now</a>.
```
Comment (frenck) [inline source/_docs/z-wave/control-panel.markdown]:
```suggestion

The current Z-Wave integration will eventually be deprecated and replaced with a [new implementation based on OpenZWave Daemon](https://developers.home-assistant.io/blog/2020/02/04/new-zwave/); it's currently in beta, and you can [try it now](/integrations/ozw/).

```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @adamjernst 👍

**Pull request merged**